src/functions.jl

```diff
@@ -116,6 +116,10 @@ function Base.mapslices(f, A::KeyedArray; dims)
     KeyedArray(data, new_keys)#, copy(A.meta))
 end
 
+function Base.selectdim(A::KeyedArray, s::Symbol, i)
+    return selectdim(A, NamedDims.dim(dimnames(A), s), i)
+end
```

Comment:
Maybe I compact this? I think `dim(A, dims)` is now safe, too:
```suggestion
Base.selectdim(A::KeyedArray, s::Symbol, i) = selectdim(A, NamedDims.dim(A, s), i)
```